src/components/FloatingHelper/FloatingHelper.tsx

```diff
@@ -17,9 +20,16 @@ export interface PopoverAdapterProps {
   content: React.ReactNode;
 }
 
-const PickedPopoverPropTypes = pick(Popover.propTypes, 'placement', 'shown', 'moveBy', 'hideDelay', 'showDelay', 'appendTo', 'timeout');
-export type PickedPopoverProps = Pick<PopoverProps,    'placement'| 'shown'| 'moveBy'| 'hideDelay'| 'showDelay'| 'appendTo' | 'timeout'>;
-export type FloatingHelperProps = PickedPopoverProps & PopoverAdapterProps;
+export interface FloatingHelperOwnProps {
+  /** Controls wether a close button will appear ot not */
+  withCloseButton?: boolean;
+  /** Width HTML attribute of the content. If a number is passed then it defaults to px. e.g width={400} => width="400px" */
+  width?: string | number;
+}
+
+const PickedPopoverPropTypes = pick(Popover.propTypes, 'placement', 'shown', 'moveBy', 'hideDelay', 'showDelay', 'appendTo', 'timeout', 'className');
+export type PickedPopoverProps = Pick<PopoverProps,    'placement'| 'shown'| 'moveBy'| 'hideDelay'| 'showDelay'| 'appendTo'| 'timeout' | 'className'>;
```

Comment:
extra spaces

Reply:
on purpose, to align with row above.

Reply:
do you really need to pick props and differentiate between `FloatingHelper` and `Popover`? `...rest` is applied to `FloatingHelperBO` which is popover anyway